programs/vote/src/vote_instruction.rs

```diff
@@ -406,6 +463,23 @@ pub fn process_instruction(
                 &signers,
             )
         }
+        VoteInstruction::UpdateVoteState(vote_state_update)
+        | VoteInstruction::UpdateVoteStateSwitch(vote_state_update, _) => {
```

Comment:
This needs a feature switch to avoid losing consensus if some validators have this update and others don't

Reply:
the idea here is that we would still be able to reach consensus in this case with a mix of the old / new vote instructions

Reply:
My meaning is that validators which upgrade their software to have this change will produce a different bank hash than validators without it if any new votes are sent in the network. It needs a feature switch